consumer-java/aws-kinesis-video-producer-sdk-canary-consumer/Makefile

```diff
@@ -0,0 +1,12 @@
+consumer: build
```

Comment:
Could you rename `consumer` to `test`? `test` is the target that the Travis script is looking for. Also, you just need to add a target to .travis.yaml to enable the CI, https://github.com/aws-samples/amazon-kinesis-video-streams-demos/blob/833cbc117e206e09f7412731bf6b054adc53da4a/.travis.yml#L17

To:

```
- env: TARGET=webrtc-c/canary
- env: TARGET=consumer-java/aws-kinesis-video-producer-sdk-canary-consumer
```

Reply:
Sure!